source/simulator/core.cc

```diff
@@ -1043,6 +1046,7 @@ namespace aspect
             {
               coupling[x.velocities[d]][x.pressure] = DoFTools::always;
               coupling[x.pressure][x.velocities[d]] = DoFTools::always;
+              coupling[x.pressure][x.pressure] = DoFTools::always;
```

Comment:
Yes, I agree that the problem is not well-posed for the reasons that you stated!

I think that the solution proposed by @tjhei in #1865 is probably the way to go. I think that if you allow that pull request (once the errors are resolved), this benchmark will be possible without further modification of the core routines. Attaching an additional signal is also fine with me.

I have seen some *interesting* convergence behavior with this benchmark. I am still working on the correct refinement criteria to deliver an acceptable match to Van Keken's values. I did align the mesh to the slab interface. One of the numbers required for the benchmark is the temperature at a specific point (tip of the wedge) and it is probably not possible to get good agreement without this point aligning with a node.